cibyl/sources/zuul/models.py

```diff
@@ -13,13 +13,20 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 """
+from typing import Dict
+
 from cibyl.models.ci.zuul.build import Build
 from cibyl.models.ci.zuul.job import Job
 from cibyl.models.ci.zuul.pipeline import Pipeline
 from cibyl.models.ci.zuul.project import Project
 from cibyl.models.ci.zuul.tenant import Tenant
 
 
+class Model(Dict[str, Tenant]):
```

Comment:
I would maybe rename this to something a bit more specific. Given that it's used as the type in some docstrings, this might be confusing with the Model class that acts as base for the CI models, particularly when we build the code documentation from the docstrings.

Reply:
Good point, I will give it another name.